content/tutorials/channel-occupancy-events.textile

```diff
@@ -91,114 +80,170 @@ Since we'll be using JavaScript in this tutorial, the best way to display the re
 
 ```[javascript]
-var apiKey = '<YOUR-API-KEY>';
+var apiKey = '<YOUR_API_KEY>';                                                                                                                                                          
 var ably = new Ably.Realtime({
     key: apiKey
 });
 ```
 
-Next, let's instance the metachannel using the realtime client that we instanced above and also reference the text area we added in the HTML, to display results in.
+Remember to replace the text '<YOUR_API_KEY>' with your Ably API key. For example:
+
+```[javascript]
+var apiKey = '{{API_KEY}}';                                                                                                                                                         
+```
+
+Next, set the channel parameters to enable inband occupancy events:
+
+```[javascript]
+var regularChannelName = "channel-" + Math.random().toString(36).substr(2, 16)
+var channelOpts = { params: { occupancy: "metrics" } };
+var channel = ably.channels.get(regularChannelName, channelOpts);
+```
+
+Then you need to reference the text area you added in the HTML:
 
 ```[javascript]
-var metaChannel = ably.channels.get("[meta]channel.lifecycle");
 var resultArea = document.getElementById("result");
 resultArea.scrollTop = resultArea.scrollHeight;
 ```
 
-The next step would be to subscribe to the channel occupancy events. This event is triggered on the @[meta]channel.lifecycle@ that we instanced above and bears the name 'channel.occupancy'. We can display the results returned in the callback in the text area in our HTML.
+This text area is where your results are displayed.
+
+You now need to subscribe to the inband channel occupancy events. You can display the results returned in the callback in the text area defined in your HTML:
 
 ```[javascript]
-metaChannel.subscribe('channel.occupancy', (msg) => {
-    var msgJSONobj = JSON.parse(JSON.stringify(msg.data));
-    //extract occupancy data from the message returned in the callback
-    var occupancyMetrics = msgJSONobj.status.occupancy.metrics
-    if (occupancyMetrics && !msgJSONobj.name.includes('[meta]')) {
-        resultArea.value += ('\n\n[METADATA - ' + (new Date().toLocaleTimeString()) + ' ]: Occupancy on channel "' + msgJSONobj.name + '" has been updated. New data is as follows:\n')
-        resultArea.value += ('Connections: ' + occupancyMetrics.connections + ' \n')
-        resultArea.value += ('Publishers: ' + occupancyMetrics.publishers + ' \n')
-        resultArea.value += ('Subscribers: ' + occupancyMetrics.subscribers + ' \n')
-        resultArea.value += ('Presence Connections: ' + occupancyMetrics.presenceConnections + ' \n')
-        resultArea.value += ('Presence Members: ' + occupancyMetrics.presenceMembers + ' \n')
-        resultArea.value += ('Presence Subscribers: ' + occupancyMetrics.presenceSubscribers + ' \n')
-        resultArea.scrollTop = resultArea.scrollHeight;
-    }
-})
+channel.subscribe("[meta]occupancy", (msg) => {
```

Comment:
pls review throughout for single vs double quotes

Reply:
Done.